xml/rmt_install.xml

```diff
@@ -219,638 +220,721 @@
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>: <literal>[]</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: List of products to
+        enable for mirroring
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: app.scc.products_disable</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: list
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>: <literal>[]</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: List of products to
+        disabled for mirroring.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: ingress.enabled</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: bool
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>: <literal>false</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: Ingress Enabled
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: ingress.hosts[0]</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: object
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>:
+        <literal>{"host":"chart-example.local","paths":[{"path":"/","pathType":"Prefix"}]}</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: DNS name at which the
+        &rmt; service will be accessible from clients
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: ingress.tls[0].hosts[0]</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: string
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>:
+        <literal>"chart-example.local"</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: DNS name at which the
+        &rmt; service will be accessible from clients
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: ingress.tls[0].secretName</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: string
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>:
+        <literal>"rmt-cert"</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: TLS Ingress
+        Certificate.
+       </para>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+   </sect3>
+   <sect3 xml:id="sec-rmt-helm-optional-values">
```

Comment:
This list is huge, and I don't think most customers will need to change any optional values.
Also, the options are already documented in the default values.yaml file (right?), so I don't see a need for documenting that here again.
I'd fully remove this section.

Reply:
@tomschr is it possible to have a collapsible list?

Reply:
Unfortunately, not with our current tool chain.

Reply:
Thanks

Reply:
Then I propose we remove the entire "optional" section.